src/Blankslate/Blankslate.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import {BookIcon} from '@primer/octicons-react'
+import {Meta, Story} from '@storybook/react'
+import React from 'react'
+import {Blankslate} from '../Blankslate'
+import {ComponentProps} from '../utils/types'
+
+export default {
+  title: 'Drafts/Components/Blankslate',
+  component: Blankslate,
```

Comment:
Could you add `'Blankslate'` to the list of components in the storybook test? https://github.com/primer/react/blob/main/src/__tests__/storybook.test.tsx